minio/credentials/parsers.py

```diff
@@ -0,0 +1,52 @@
+from datetime import datetime
```

Comment:
One change we need to add copyright header.

Reply:
On it